CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 Changelog
 =========
 
+## [1.20.1] - 2020-09-24
+### Fixed
+ * Set ConnContext in the channel instead of the connection (#806)
```

Comment:
add some context on why, e.g., "to fix marshalling of connection options, often used in configurations" or something